File: auto-docs/executables/julia/Greens_heatmap.jl
using Plotly

using Plotly
Plotly.signin("theengineear", "o9zlr0hy6z")

size = 50
z = rand(size, size)
for r = 1:size
  for c = 1:size
        z(r,c) = sqrt(r*c/size^2)
data = [
  [
    "z" => z, 
    "scl" => "Greens", 
    "type" => "heatmap"
  ]
]
layout = ["title" => "Greens"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "Greens-heatmap", "fileopt" => "overwrite", "auto_open" => "false"])
plot_url = response["url"]

File: auto-docs/executables/python/Greens_heatmap.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')

import numpy as np

size = 50
z = np.zeros((size, size)).tolist()
for r in range(1, size + 1):
    for c in range(1, size + 1):
        z[r-1][c-1] = r+c

data = Data([
    Heatmap(
        z=z,
        scl='Greens'
    )
])
layout = Layout(
    title='Greens'
)
fig = Figure(data=data, layout=layout)

plot_url = py.plot(fig, filename='Greens-heatmap', auto_open=False)